Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -43,7 +44,7 @@ public class AlertStore {
         let storeDescription = NSPersistentStoreDescription()
         if let storageDirectoryURL = storageDirectoryURL {
             let storageFileURL = storageDirectoryURL
-                .appendingPathComponent("AlertStore")
+                .appendingPathComponent(AlertStore.storageDirectoryPathComponent)
```

Comment:
[nit] Making the location of the file backing this store solely a concern of the caller might be a little cleaner.

Reply:
I considered that.  I am trying to avoid having the store need to make the directory for placement.

Reply:
Yeah, the store would not need to worry about that if we make the storage location a concern of the caller. See 
https://github.com/tidepool-org/Loop/pull/229/commits/1d9791a263bc706391565ea4a223322e64d5b769

Reply:
Oh, I see.  If I understand you correctly, you are not a fan of imbuing `AlertStore` with the knowledge of what subdirectory to live in.  That's fair.  I'll change that.